session2/nmt.py

```diff
@@ -250,6 +250,39 @@ def param_init_gru(options, params, prefix='gru', nin=None, dim=None):
     return params
 
 
+def param_init_gru_nonlin(options, params, prefix='gru',
```

Comment:
How about defining ``U_nl``, ``b_nl``, ``Ux_nl`` and ``bx_nl`` in ``param_init_gru_cond`` after calling ``param_init_gru``, instead of making a new function ``param_init_gru_nonlin``?